data/test-by-folder.json

```
@@ -3284,6 +3284,17 @@
       "Mobile V2: Messaging - Save and Unsave Message"
     ]
   },
+  {
+    "folder": "mobile-v2/mobile-draft",
+    "tests": [
+      "Create Scheduled message in a Channel.",
+      "Deleting a scheduled messages in mobile.",
+      "Edit Scheduled message created.",
+      "Rescheduled messages in mobile.",
+      "Sending scheduled messages in mobile.",
```

Are we sending multiple messages or just one?

```suggestion
      "Sending scheduled messages.",
```